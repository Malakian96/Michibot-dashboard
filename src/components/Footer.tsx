const Footer = () => {
  return (
    <footer className="w-full p-4 bg-discord-dark-light text-center text-sm text-discord-gray">
      Â© 2024 Michibot. All rights reserved.
    </footer>
  );
};

export default Footer;
