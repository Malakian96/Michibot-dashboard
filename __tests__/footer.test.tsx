import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../src/components/Footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const { getByText } = render(<Footer />);
    expect(
      getByText('Â© 2024 Michibot. All rights reserved.'),
    ).toBeInTheDocument();
  });
});

describe('FooterSnapshot', () => {
  it('matches snapshot', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
