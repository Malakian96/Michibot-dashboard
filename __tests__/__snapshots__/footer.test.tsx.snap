// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSnapshot matches snapshot 1`] = `
<div>
  <footer
    class="w-full p-4 bg-discord-dark-light text-center text-sm text-discord-gray"
  >
    Â© 2024 Michibot. All rights reserved.
  </footer>
</div>
`;
